docs/releases/5.0.txt

```diff
@@ -387,6 +387,10 @@ Miscellaneous
   :class:`~django.db.models.BaseConstraint` is deprecated in favor of
   keyword-only arguments.
 
+* ``request`` is added to the signature of
+  :meth:`~django.contrib.admin.ModelAdmin.lookup_allowed()`. Support  for
```

Comment:
`()` is unnecessary for `:meth:` and `:func:` directives, as it's added automatically by Sphinx.